src/Engine/ProtoImperative/ProtoImperative.csproj

```diff
@@ -78,6 +82,12 @@
   <PropertyGroup>
     <AssemblyOriginatorKeyFile>ProtoImperative.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+    <Prefer32Bit>false</Prefer32Bit>
+  </PropertyGroup>
   <ItemGroup>
```

Comment:
Let's remove all of these `Prefer32Bit` properties from this PR.  They are highly confusing.